weed/topology/store_replicate.go

```diff
@@ -43,7 +44,9 @@ func ReplicatedWrite(masterFn operation.GetMasterFn, grpcDialOption grpc.DialOpt
 	}
 
 	if s.GetVolume(volumeId) != nil {
+		start := time.Now()
 		isUnchanged, err = s.WriteVolumeNeedle(volumeId, n, true, fsync)
+		stats.VolumeServerRequestHistogram.WithLabelValues("localWrite").Observe(time.Since(start).Seconds())
```

Comment:
actually, I think moving to constants is better. And align the naming there, similar to stats.ErrorWriteToLocalDisk